### PR TITLE
add team docs as codeowners of clerk-docs

### PR DESCRIPTION
**Pull request by alexisintech**
adds team docs as codeowners to all files (*) in the docs repo

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @clerk/docs
```
Comment (kylemac) [inline .github/CODEOWNERS:1]: @alexisintech ever notice these little 🔴  red circles in your diff view? github expects a new line at the end of many file types and thats why it's showing that. you can add this to your vscode settings and it'll just do it onSave for ya without ever having to think about it and then poof, no more annoying 🔴 circles on yo PRs... 

https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files

File: .gitignore [hunk 2]
```diff
@@ -42,3 +42,6 @@ yarn.lock
 
 /docs/docs
 /docs/docs/
+
+# CODEOWNERS
+/.github/CODEOWNERS
```
Comment (kylemac) [inline .gitignore:47]: @alexisintech why do we want CODEOWNERS in our `.gitignore`?

File: .github/CODEOWNERS (hunk 1 above)
Comment (JacobMGEvans) [reply .github/CODEOWNERS:1]: Ive brought this up before, there needs to be a Prettier config or .editorconfig that handles this for everyone.

File: .gitignore (hunk 2 above)
Comment (alexisintech) [reply .gitignore]: tbh i have no clue why i did that today

File: .github/CODEOWNERS (hunk 1 above)
Comment (alexisintech) [reply .github/CODEOWNERS:1]: woah fireee.
and yeah @JacobMGEvans adding prettier is something we want to add, but when i previously added it, it was messing up our custom components and i was really busy and didn't have time to dig and figure out how to get it to ignore our custom components so i just stripped prettier for the time being

**Review by kylemac — APPROVED** [review]

**Pull request merged**